chain_name: larm
chain_root: HeadPitch_link
chain_tip: l_wrist
marker_type: checkerboard       # checkerboard, aruco, circle, color
marker_frame: LMarker_frame

# checkerboard specific parameters
checkerboard_rows: 4            # optional; if not set, default is 4
checkerboard_columns: 4         # optional; if not set, default is 4